content/blog/setting-up-github-actions-for-v.md

```diff
@@ -130,12 +139,28 @@ Broadly, here are the steps we need to perform:
 6. Run the unit tests if any.
 7. Optionally upload the built artifacts to be used later.
 
+> Artifacts are files that are produced by the build process. They can be
+> executables, logs, results, trained models, or any other file that has a
+> significance to the end user, and can be reused after the pipeline has
+> finished.
+>
+> Refer to the [documentation](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts)
+> for more information.
+
 The reference workflow files are available here:
 [workflows](https://github.com/hungrybluedev/geo/tree/main/.github/workflows).
 
 ## Setting Up Github Actions CI for V
 
 First we create a new file `ci.yml` in the `.github/workflows` directory.
+We have a few options to choose from:
```

Comment:
Hmm... not sure this section is needed - it seems incongruously basic, compared to the rest of the article.

Reply:
I wrote this in response to this question:

> How does one get to the .github/workflows directory, to create a new file? I did it by using the "Actions" menu on the github page, and then configuring their Simple workflow. Is there another way I don't know about?

Maybe I can shorten the explanation here.